client/tinychain/ml/mlr.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Sun Feb  6 16:59:54 2022
+@author: meracoda
+"""
+
+from tinychain.collection.tensor import einsum, Dense
+from tinychain.ml import Layer, NeuralNet
+from tinychain.ref import After
+import numpy as np
+
+
+class MultipleLinearRegression :
+    
+    @property
+    def learningrate(self):  
+        '''
+        We use property accessor so we recall self.learningrate instead of Dense(self[0])
+        '''
+        return Dense(self[0])
+    @property
+    def iteration(self):  
+        '''
+        We use property accessor so we recall self.iteration instead of Dense(self[1])
+        '''
+        return Dense(self[1])
+    
+    def load (self):  # Question  i need to instanitiate my class with this 
+                                    # i guess i need to add @class_method ?
+        return MultipleLinearRegression([learningrate,iteration])
+    
+    def predict(self, inputs, weights, learning = True):
+        if learning==True:
+            preds = tc.tensor.einsum("ij,ki->kj", [weights, inputs]) 
+        else:
+            inputs = np.insert(inputs ,0,1,axis=1)   # i want to add column of ones at index 0
+            # inputs = tc.concatenate(inputs,0 , axis =1) how about this ?
+            preds = tc.tensor.einsum("ij,ki->kj", [weights, inputs])            
+        return preds
+    
+    def cost_func( self, labels ,preds):
+        # is it okay to use numpy.sum ?
+        cost = np.sum((labels - preds)**2)/len(labels)
```

Comment:
no, `labels` and `preds` will be TinyChain `Tensor`s which NumPy won't know what to do with